.ci/jobs/e2e-testing-mbp.yml

```diff
@@ -13,8 +13,11 @@
     script-path: .ci/Jenkinsfile
     scm:
       - github:
+          property-strategies:
+            all-branches:
+              - suppress-scm-triggering: true
           branch-discovery: no-pr
-          head-filter-regex: '(main|PR-.*|8\.\d+|7\.17|feature-.*)'
+          head-filter-regex: '(main|8\.\d+|7\.17|feature-.*)'
```

Comment:
this change the current behaviour on the stable branch without a change in the pipelines, this breaks the CI.

Reply:
A proposal here, do not touch the main pipeline (.ci/jobs/e2e-testing-mbp.yml) until everything is working. So, create a new file like this one with a different `job.name`, and use it until you have a process refined and working. For that, you will have to limit the `head-filter-regex` to a specific branch(where you are working) and PR (only your test PRs). then we can see which changes we can incorporate on the main branch without issues.

Reply:
WYT @v1v

Reply:
@kuisathaverat Great solution! Could you please take a look now?

Reply:
we still making changes in .ci/jobs/e2e-testing-mbp.yml that is something we do not want